src/models/document/document.ts

```diff
@@ -48,6 +49,7 @@ export const DocumentModel = Tree.named("Document")
   })
   .volatile(self => ({
     queryPromise: undefined as Promise<UseQueryResult<IGetNetworkDocumentResponse>> | undefined,
+    instanceId: nanoid()
```

Comment:
We added this for debugging. I think you should remove it, to prevent confusion.